qlib/CsvUtil.qm

```diff
@@ -657,6 +657,9 @@ public namespace CsvUtil {
             # list of idx to field transformarions, in order of spec
             hash m_resolve_by_idx;
 
+            # fake specs based on the first non-header row
+            bool fakeHeaderNames;
```

Comment:
can you fix the naming differences, please? there is m_resolve_by_idx, but fakeHeaderNames member...